src/Model/Item.php

```diff
@@ -2046,6 +2046,8 @@ public static function guidFromUri(string $uri, string $host = null): string
 		// Remove the scheme to make sure that "https" and "http" doesn't make a difference
 		unset($parsed['scheme']);
 
+		$host = $host ?? $parse['host'] ?? 'example.com';
```

Comment:
`example.com`?

Reply:
Why not? When the host was missing before it would have been using empty string which is less fun.

Reply:
I would prefer some logging with all parameters and the callstack, since there *should* be a resolvable host name provided at any time and just use `''` when the host was not provided.

Reply:
Ok. I still stand by the fact that it's less fun.

Reply:
Yeah, it is, sorry for being German :grin: